packages/patternlab/styleguide/source/assets/js/modules/accordions.js

```diff
@@ -3,11 +3,19 @@ import checkActive from "../helpers/cssControlCode.js";
 export default (function (window,document,$,undefined) {
   let $jsAccordion = $('.ma__collapsible-content .js-accordion');
   var $toggleLink = $('.ma__collapsible-content__toggle-all');
+  let accordionIds = "";
 
   $jsAccordion.each(function(index){
     init.apply(this, [index]);
   });
 
+  // Set up aria label for collapse / expand button.
+  $('.ma__collapsible-content--extended .js-accordion').each(function(index){
+    accordionIds += 'accordion' + (index + 1) + ' ';
```

Comment:
`accordion1`, `accordion2`, ...  are just fallback IDs right? 
See 
```
id = $content.attr('id') || 'accordion' + (index + 1),
```

Reply:
to be accurate, we should probably use `$content.attr('id')`

Reply:
sorry, what you mean with fallback ids? accordion1, accordion2, etc are applied to each individual accordion, see line 112 `$content.attr('id', id);`

Reply:
see the line above this line 
```
id = $content.attr('id') || 'accordion' + (index + 1),
```

Reply:
Just updated the logic so it uses the existing id if the element has one, or generates a new one otherwise as it is here:
 `id = $content.attr('id') || 'accordion' + (index + 1)`

I also updated the aria label according to what we chatted on Teams.